app/google-services.json

```diff
@@ -0,0 +1,69 @@
+{
+  "project_info": {
+    "project_number": "409851798978",
+    "firebase_url": "https://oppia-android-40f74.firebaseio.com",
+    "project_id": "oppia-android-40f74",
```

Comment:
Just to confirm, this is for testing purposes only right? Maybe its worth adding a comment to mention this is our test project (@BenHenning FYI when we actually release, we would want to change these IDs to the project you are creating)

Reply:
Also, @Sarthak2601 can you comment on all parts of this json file that would need an update when we have to release (I know we would have to change project_id, and maybe storage_bucket..what about the client params below?)

Reply:
I think we should replace the whole json file at the time of release because almost everything will change.

Reply:
We probably shouldn't check this in until it's associated with our actual project.

Further, is there anything we need to be worried about regarding exposing the information here to the world? Ie, does any of this need to be private? If so, we probably need to create two different projects: one that developers use, and one that's used by the shipped app.

Reply:
The problem is that the app won't build if this file doesn't exist. These params are dummy for the test world, what we could do is just keep an empty or minimal json file just so that the app builds

Reply:
This has been moved to another PR, resolving this